TechnicoServer/TechnicoDomain/Program.cs

```diff
@@ -0,0 +1,6 @@
+﻿using TechnicoDomain.Context;
```

Comment:
This "Program.cs" file is not used in the Project, you could delete it from your repostitory.